src/library-authoring/LibraryBlock/LibraryBlock.tsx

```diff
@@ -7,6 +7,7 @@ import messages from './messages';
 interface LibraryBlockProps {
   onBlockNotification?: (event: { eventType: string; [key: string]: any }) => void;
   usageKey: string;
+  version?: 'published' | 'draft' | number;
```

Comment:
Worth defining this as a type so it can be re-used in the [CompareChangesWidget](https://github.com/openedx/frontend-app-authoring/pull/1393/files#diff-6ac252bc6fca225c23323006d128f41c7f8ade489057293030006cde45e2e658)?

Reply:
Good idea. Done.